web/src/routes/(user)/map/[[photos=photos]]/[[assetId=id]]/+page.svelte

```diff
@@ -113,14 +115,16 @@
   >
   <Portal target="body">
     {#if $showAssetViewer}
-      <AssetViewer
-        asset={$viewingAsset}
-        showNavigation={viewingAssets.length > 1}
-        on:next={navigateNext}
-        on:previous={navigatePrevious}
-        on:close={() => assetViewingStore.showAssetViewer(false)}
-        isShared={false}
-      />
+      {#await import('../../../../../lib/components/asset-viewer/asset-viewer.svelte') then AssetViewer}
```

Comment:
Can we... not do that? lol
A $lib import would be much nicer here haha. Also, why do even need to import it inline?

Reply:
I believe it was imported via this await block in order to speed initial page load